### ISSUE TITLE
Version 0.6.0 crashes on linux with nvidia proprietary driver (version 0.5.1 works fine)

### ISSUE THREAD

**Issue by ZaWertun** [flightlessmango/MangoHud#412]
After update to 0.6.0 I see crash every time I try to run program with mangohud:
```
$ mangohud vkcube
skipping config: /usr/bin/MangoHud.conf [ not found ]
skipping config: /home/zawertun/.config/MangoHud/vkcube.conf [ not found ]
skipping config: /home/zawertun/.config/MangoHud/MangoHud.conf [ not found ]
fish: “mangohud vkcube” terminated by signal SIGSEGV (Address boundary error)
```

Same crash when running `glxgears`.

**Comment by ZaWertun** [comment]
I have NVidia driver version 455.45.01 installed.
OS: Fedora 33 x86_64.

**Comment by Leopard1907** [comment]
I can't repro it here. Linux Mint 20 , same NV driver.

![Ekran Görüntüsü - 2020-11-29 17-02-23](https://user-images.githubusercontent.com/24878978/100544270-8fc3ed80-3265-11eb-8802-26d8d68520ef.png)
![Ekran Görüntüsü - 2020-11-29 17-02-50](https://user-images.githubusercontent.com/24878978/100544273-905c8400-3265-11eb-8a6d-31984b8a87be.png)

**Comment by ZaWertun** [comment]
I'll try to submit backtrace, still waiting for debuginfo packages to install.
Also Fedora packages is built with LTO enabled, don't think Mint packages was built with it.

**Comment by flgx16** [comment]
What's your MangoHud config?

**Comment by ZaWertun** [comment]
> What's your MangoHud config?

I have no config, just default.

**Comment by Leopard1907** [comment]
I used Github release, not any repo package. Can you try with release at least?

https://github.com/flightlessmango/MangoHud/releases/tag/v0.6.0

**Comment by ZaWertun** [comment]
> I used Github release, not any repo package. Can you try with release at least?
> 
> https://github.com/flightlessmango/MangoHud/releases/tag/v0.6.0

Just removed mangohud installed with package manager and installed from tar.gz - still got crash.

**Comment by ZaWertun** [comment]
Backtrace:
```c++
#0  0x00007fcc0fcf57bb in _IO_new_fclose (fp=0x346e6f6d77682f) at iofclose.c:48
48      iofclose.c: No such file or directory.
Thread 1 (Thread 0x7fcc1001b740 (LWP 34227)):
#0  0x00007fcc0fcf57bb in _IO_new_fclose (fp=0x346e6f6d77682f) at iofclose.c:48
status = <optimized out>
#1  0x00007fcc100dccae in CPUPowerData_k10temp::~CPUPowerData_k10temp (this=<optimized out>, this=<optimized out>) at ../src/cpu.h:64
No locals.
#2  init_cpu_power_data_k10temp (path="/sys/class/hwmon/hwmon2") at ../src/cpu.cpp:519
powerData = 0x55ac453f31c0
coreCurrentInput = ""
socVoltageInput = ""
coreVoltageInput = ""
socCurrentInput = ""
success = <optimized out>
error = <optimized out>
powerData = <optimized out>
coreVoltageInput = Python Exception <class 'gdb.error'> value has been optimized out: 
coreCurrentInput = Python Exception <class 'gdb.error'> value has been optimized out: 
socVoltageInput = Python Exception <class 'gdb.error'> value has been optimized out: 
socCurrentInput = Python Exception <class 'gdb.error'> value has been optimized out: 
#3  CPUStats::InitCpuPowerData() [clone .constprop.0] (this=<optimized out>) at ../src/cpu.cpp:585
dir = ""
__for_range = std::vector of length 5, capacity 8 = {"hwmon4", "hwmon2", "hwmon0", "hwmon3", "hwmon1"}
__for_begin = Python Exception <class 'gdb.error'> value has been optimized out: 
__for_end = Python Exception <class 'gdb.error'> value has been optimized out: 
name = "k10temp"
path = "/sys/class/hwmon/hwmon2"
hwmon = "/sys/class/hwmon/"
cpuPowerData = 0x0
dirs = std::vector of length 5, capacity 8 = {"hwmon4", "hwmon2", "hwmon0", "hwmon3", "hwmon1"}
#4  0x00007fcc100880ad in init_cpu_stats (params=...) at ../src/vulkan.cpp:465
enabled = @0x55ac453f66d8: {true, true, false, false, false, false, true, true, false <repeats 22 times>, true, false, true}
#5  0x00007fcc1009b4ef in overlay_CreateInstance (pCreateInfo=<optimized out>, pAllocator=<optimized out>, pInstance=<optimized out>) at ../src/vulkan.cpp:2147
chain_info = <optimized out>
engineName = "VULKAN"
engineVersion = ""
fpGetInstanceProcAddr = <optimized out>
fpCreateInstance = <optimized out>
result = VK_SUCCESS
instance_data = 0x55ac453f64c0
#6  0x00007fcc0f86ab6c in device_select_CreateInstance (pCreateInfo=0x7fffdf646400, pAllocator=0x0, pInstance=0x7fffdf6464d8) at ../src/vulkan/device-select-layer/device_select_layer.c:146
chain_info = 0x7fffdf646340
info = 0x55ac45104710
fpCreateInstance = <optimized out>
result = <optimized out>
#7  0x00007fcc0fe70039 in loader_create_instance_chain (pCreateInfo=<optimized out>, pAllocator=<optimized out>, inst=0x55ac451ff9f0, created_instance=<optimized out>) at ../loader/loader.c:6051
instance_dispatch = {sType = VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO, pNext = 0x7fffdf646340, function = VK_LOADER_DATA_CALLBACK, u = {pLayerInfo = 0x7fcc0fe66120 <vkSetInstanceDispatch>, pfnSetInstanceLoaderData = 0x7fcc0fe66120 <vkSetInstanceDispatch>, layerDevice = {pfnLayerCreateDevice = 0x7fcc0fe66120 <vkSetInstanceDispatch>, pfnLayerDestroyDevice = 0x0}, loaderFeatures = 266756384}}
device_callback = {sType = VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO, pNext = 0x7fffdf6463d0, function = VK_LOADER_LAYER_CREATE_DEVICE_CALLBACK, u = {pLayerInfo = 0x7fcc0fe6d1f0 <loader_layer_create_device>, pfnSetInstanceLoaderData = 0x7fcc0fe6d1f0 <loader_layer_create_device>, layerDevice = {pfnLayerCreateDevice = 0x7fcc0fe6d1f0 <loader_layer_create_device>, pfnLayerDestroyDevice = 0x7fcc0fe65680 <loader_layer_destroy_device>}, loaderFeatures = 266785264}}
loader_features = {sType = VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO, pNext = 0x7fffdf6463a0, function = VK_LOADER_FEATURES, u = {pLayerInfo = 0x0, pfnSetInstanceLoaderData = 0x0, layerDevice = {pfnLayerCreateDevice = 0x0, pfnLayerDestroyDevice = 0x0}, loaderFeatures = 0}}
activated_layers = <optimized out>
chain_info = {sType = VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO, pNext = 0x0, function = VK_LAYER_LINK_INFO, u = {pLayerInfo = 0x0, pfnSetInstanceLoaderData = 0x0, layerDevice = {pfnLayerCreateDevice = 0x0, pfnLayerDestroyDevice = 0x7fffdf646430}, loaderFeatures = 0}}
layer_instance_link_info = <optimized out>
loader_create_info = {sType = VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO, pNext = 0x7fffdf646370, flags = 0, pApplicationInfo = 0x7fffdf6466b0, enabledLayerCount = 0, ppEnabledLayerNames = 0x7fffdf646940, enabledExtensionCount = 2, ppEnabledExtensionNames = 0x7fffdf646fe0}
res = <optimized out>
next_gipa = 0x7fcc0f86ad60 <get_instance_proc_addr>
cur_gipa = <optimized out>
cur_gdpa = <optimized out>
next_gpdpa = <optimized out>
cur_gpdpa = <optimized out>
feature_flags = 0
fpCreateInstance = <optimized out>
#8  0x00007fcc0fe7fe6c in vkCreateInstance (pCreateInfo=<optimized out>, pAllocator=<optimized out>, pInstance=<optimized out>) at ../loader/trampoline.c:519
ptr_instance = <optimized out>
created_instance = 0x55ac451ff9f0
loaderLocked = <optimized out>
res = VK_SUCCESS
ici = {sType = VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO, pNext = 0x0, flags = 0, pApplicationInfo = 0x7fffdf6466b0, enabledLayerCount = 0, ppEnabledLayerNames = 0x7fffdf646940, enabledExtensionCount = 2, ppEnabledExtensionNames = 0x7fffdf646fe0}
out = <optimized out>
#9  0x000055ac44b2c643 in demo_init_vk (demo=0x7fffdf6469c0) at ../cube/cube.c:3155
err = <optimized out>
instance_extension_count = 18
instance_layer_count = 0
instance_validation_layers = {0x55ac44b3346a "VK_LAYER_KHRONOS_validation"}
validation_found = <optimized out>
inst_info = {sType = VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO, pNext = 0x0, flags = 0, pApplicationInfo = 0x7fffdf6466b0, enabledLayerCount = 0, ppEnabledLayerNames = 0x7fffdf646940, enabledExtensionCount = 2, ppEnabledExtensionNames = 0x7fffdf646fe0}
gpu_count = 4
device_extension_count = 896
swapchainExtFound = <optimized out>
surfaceExtFound = 1
app = {sType = VK_STRUCTURE_TYPE_APPLICATION_INFO, pNext = 0x0, pApplicationName = 0x55ac44b33460 "vkcube", applicationVersion = 0, pEngineName = 0x55ac44b33460 "vkcube", engineVersion = 0, apiVersion = 4194304}
dbg_messenger_create_info = {sType = VK_STRUCTURE_TYPE_PIPELINE_VERTEX_INPUT_STATE_CREATE_INFO, pNext = 0x7fcc1005f3d0, flags = 12, messageSeverity = 0, messageType = 0, pfnUserCallback = 0x7fffdf646730, pUserData = 0x0}
platformSurfaceExtFound = 1
physDevFeatures = {robustBufferAccess = 1158664880, fullDrawIndexUint32 = 21932, imageCubeArray = 1, independentBlend = 112, geometryShader = 1, tessellationShader = 0, sampleRateShading = 3747899200, dualSrcBlend = 32767, logicOp = 0, multiDrawIndirect = 32716, drawIndirectFirstInstance = 0, depthClamp = 0, depthBiasClamp = 3747899768, fillModeNonSolid = 32767, depthBounds = 16416, wideLines = 0, largePoints = 3747899696, alphaToOne = 32767, multiViewport = 3, samplerAnisotropy = 32767, textureCompressionETC2 = 6, textureCompressionASTC_LDR = 0, textureCompressionBC = 0, occlusionQueryPrecise = 3, pipelineStatisticsQuery = 268411143, vertexPipelineStoresAndAtomics = 32716, fragmentStoresAndAtomics = 0, shaderTessellationAndGeometryPointSize = 32767, shaderImageGatherExtended = 268824368, shaderStorageImageExtendedFormats = 32716, shaderStorageImageMultisample = 270028960, shaderStorageImageReadWithoutFormat = 32716, shaderStorageImageWriteWithoutFormat = 2066, shaderUniformBufferArrayDynamicIndexing = 0, shaderSampledImageArrayDynamicIndexing = 4210031, shaderStorageBufferArrayDynamicIndexing = 0, shaderStorageImageArrayDynamicIndexing = 2066, shaderClipDistance = 0, shaderCullDistance = 269917164, shaderFloat64 = 32716, shaderInt64 = 1, shaderInt16 = 0, shaderResourceResidency = 2147483648, shaderResourceMinLod = 0, sparseBinding = 0, sparseResidencyBuffer = 0, sparseResidencyImage2D = 269884077, sparseResidencyImage3D = 32716, sparseResidency2Samples = 1, sparseResidency4Samples = 0, sparseResidency8Samples = 266717932, sparseResidency16Samples = 32716, sparseResidencyAliased = 0, variableMultisampleRate = 0, inheritedQueries = 269884077}
err = <optimized out>
instance_extension_count = <optimized out>
instance_layer_count = <optimized out>
instance_validation_layers = {<optimized out>}
validation_found = <optimized out>
surfaceExtFound = <optimized out>
platformSurfaceExtFound = <optimized out>
app = {sType = <optimized out>, pNext = <optimized out>, pApplicationName = <optimized out>, applicationVersion = <optimized out>, pEngineName = <optimized out>, engineVersion = <optimized out>, apiVersion = <optimized out>}
inst_info = {sType = <optimized out>, pNext = <optimized out>, flags = <optimized out>, pApplicationInfo = <optimized out>, enabledLayerCount = <optimized out>, ppEnabledLayerNames = <optimized out>, enabledExtensionCount = <optimized out>, ppEnabledExtensionNames = <optimized out>}
dbg_messenger_create_info = {sType = <optimized out>, pNext = <optimized out>, flags = <optimized out>, messageSeverity = <optimized out>, messageType = <optimized out>, pfnUserCallback = <optimized out>, pUserData = <optimized out>}
gpu_count = <optimized out>
device_extension_count = <optimized out>
swapchainExtFound = <optimized out>
physDevFeatures = {robustBufferAccess = <optimized out>, fullDrawIndexUint32 = <optimized out>, imageCubeArray = <optimized out>, independentBlend = <optimized out>, geometryShader = <optimized out>, tessellationShader = <optimized out>, sampleRateShading = <optimized out>, dualSrcBlend = <optimized out>, logicOp = <optimized out>, multiDrawIndirect = <optimized out>, drawIndirectFirstInstance = <optimized out>, depthClamp = <optimized out>, depthBiasClamp = <optimized out>, fillModeNonSolid = <optimized out>, depthBounds = <optimized out>, wideLines = <optimized out>, largePoints = <optimized out>, alphaToOne = <optimized out>, multiViewport = <optimized out>, samplerAnisotropy = <optimized out>, textureCompressionETC2 = <optimized out>, textureCompressionASTC_LDR = <optimized out>, textureCompressionBC = <optimized out>, occlusionQueryPrecise = <optimized out>, pipelineStatisticsQuery = <optimized out>, vertexPipelineStoresAndAtomics = <optimized out>, fragmentStoresAndAtomics = <optimized out>, shaderTessellationAndGeometryPointSize = <optimized out>, shaderImageGatherExtended = <optimized out>, shaderStorageImageExtendedFormats = <optimized out>, shaderStorageImageMultisample = <optimized out>, shaderStorageImageReadWithoutFormat = <optimized out>, shaderStorageImageWriteWithoutFormat = <optimized out>, shaderUniformBufferArrayDynamicIndexing = <optimized out>, shaderSampledImageArrayDynamicIndexing = <optimized out>, shaderStorageBufferArrayDynamicIndexing = <optimized out>, shaderStorageImageArrayDynamicIndexing = <optimized out>, shaderClipDistance = <optimized out>, shaderCullDistance = <optimized out>, shaderFloat64 = <optimized out>, shaderInt64 = <optimized out>, shaderInt16 = <optimized out>, shaderResourceResidency = <optimized out>, shaderResourceMinLod = <optimized out>, sparseBinding = <optimized out>, sparseResidencyBuffer = <optimized out>, sparseResidencyImage2D = <optimized out>, sparseResidencyImage3D = <optimized out>, sparseResidency2Samples = <optimized out>, sparseResidency4Samples = <optimized out>, sparseResidency8Samples = <optimized out>, sparseResidency16Samples = <optimized out>, sparseResidencyAliased = <optimized out>, variableMultisampleRate = <optimized out>, inheritedQueries = <optimized out>}
instance_layers = <optimized out>
instance_extensions = <optimized out>
i = <optimized out>
physical_devices = <optimized out>
device_extensions = <optimized out>
i = <optimized out>
i = <optimized out>
i = <optimized out>
#10 demo_init (argv=<optimized out>, argc=<optimized out>, demo=<optimized out>) at ../cube/cube.c:3797
eye = {0, 3, 5}
origin = {0, 0, 0}
up = {<optimized out>, <optimized out>, <optimized out>}
eye = {<optimized out>, <optimized out>, <optimized out>}
origin = {<optimized out>, <optimized out>, <optimized out>}
up = {<optimized out>, <optimized out>, <optimized out>}
i = <optimized out>
message = <optimized out>
length = <optimized out>
usage = <optimized out>
#11 main (argc=<optimized out>, argv=<optimized out>) at ../cube/cube.c:4010
demo = {display = 0x0, connection = 0x55ac450fe2a0, screen = 0x55ac450fd2b0, xcb_window = 0, atom_wm_delete_window = 0x0, surface = 0x0, prepared = false, use_staging_buffer = false, separate_present_queue = false, is_minimized = false, gpu_number = 0, VK_KHR_incremental_present_enabled = false, VK_GOOGLE_display_timing_enabled = false, syncd_with_actual_presents = false, refresh_duration = 0, refresh_duration_multiplier = 0, target_IPD = 0, prev_desired_present_time = 0, next_present_id = 0, last_early_id = 0, last_late_id = 0, inst = 0x0, gpu = 0x0, device = 0x0, graphics_queue = 0x0, present_queue = 0x0, graphics_queue_family_index = 0, present_queue_family_index = 0, image_acquired_semaphores = {0x0, 0x0}, draw_complete_semaphores = {0x0, 0x0}, image_ownership_semaphores = {0x0, 0x0}, gpu_props ={apiVersion = 0, driverVersion = 0, vendorID = 0, deviceID = 0, deviceType = VK_PHYSICAL_DEVICE_TYPE_OTHER, deviceName = '\000' <repeats 255 times>, pipelineCacheUUID = '\000' <repeats 15 times>, limits = {maxImageDimension1D = 0, maxImageDimension2D = 0, maxImageDimension3D = 0, maxImageDimensionCube = 0, maxImageArrayLayers = 0, maxTexelBufferElements = 0, maxUniformBufferRange = 0, maxStorageBufferRange = 0, maxPushConstantsSize = 0, maxMemoryAllocationCount = 0, maxSamplerAllocationCount = 0, bufferImageGranularity = 0, sparseAddressSpaceSize = 0, maxBoundDescriptorSets = 0, maxPerStageDescriptorSamplers = 0, maxPerStageDescriptorUniformBuffers = 0, maxPerStageDescriptorStorageBuffers = 0, maxPerStageDescriptorSampledImages = 0, maxPerStageDescriptorStorageImages = 0, maxPerStageDescriptorInputAttachments = 0, maxPerStageResources = 0, maxDescriptorSetSamplers = 0, maxDescriptorSetUniformBuffers = 0, maxDescriptorSetUniformBuffersDynamic = 0, maxDescriptorSetStorageBuffers = 0, maxDescriptorSetStorageBuffersDynamic = 0, maxDescriptorSetSampledImages = 0, maxDescriptorSetStorageImages = 0, maxDescriptorSetInputAttachments = 0, maxVertexInputAttributes = 0, maxVertexInputBindings = 0, maxVertexInputAttributeOffset = 0, maxVertexInputBindingStride = 0, maxVertexOutputComponents = 0, maxTessellationGenerationLevel = 0, maxTessellationPatchSize = 0, maxTessellationControlPerVertexInputComponents = 0, maxTessellationControlPerVertexOutputComponents = 0, maxTessellationControlPerPatchOutputComponents = 0, maxTessellationControlTotalOutputComponents = 0, maxTessellationEvaluationInputComponents = 0, maxTessellationEvaluationOutputComponents = 0, maxGeometryShaderInvocations = 0, maxGeometryInputComponents = 0, maxGeometryOutputComponents = 0, maxGeometryOutputVertices = 0, maxGeometryTotalOutputComponents = 0, maxFragmentInputComponents = 0, maxFragmentOutputAttachments = 0, maxFragmentDualSrcAttachments = 0, maxFragmentCombinedOutputResources = 0, maxComputeSharedMemorySize = 0, maxComputeWorkGroupCount = {0, 0, 0}, maxComputeWorkGroupInvocations = 0, maxComputeWorkGroupSize = {0, 0, 0}, subPixelPrecisionBits = 0, subTexelPrecisionBits = 0, mipmapPrecisionBits = 0, maxDrawIndexedIndexValue = 0, maxDrawIndirectCount = 0, maxSamplerLodBias = 0, maxSamplerAnisotropy = 0, maxViewports = 0, maxViewportDimensions = {0, 0}, viewportBoundsRange = {0, 0}, viewportSubPixelBits = 0, minMemoryMapAlignment = 0, minTexelBufferOffsetAlignment = 0, minUniformBufferOffsetAlignment = 0, minStorageBufferOffsetAlignment = 0, minTexelOffset = 0, maxTexelOffset = 0, minTexelGatherOffset = 0, maxTexelGatherOffset = 0, minInterpolationOffset = 0, maxInterpolationOffset = 0, subPixelInterpolationOffsetBits =0, maxFramebufferWidth = 0, maxFramebufferHeight = 0, maxFramebufferLayers = 0, framebufferColorSampleCounts = 0, framebufferDepthSampleCounts = 0, framebufferStencilSampleCounts = 0, framebufferNoAttachmentsSampleCounts = 0, maxColorAttachments = 0, sampledImageColorSampleCounts = 0, sampledImageIntegerSampleCounts = 0, sampledImageDepthSampleCounts = 0, sampledImageStencilSampleCounts = 0, storageImageSampleCounts = 0, maxSampleMaskWords = 0, timestampComputeAndGraphics = 0, timestampPeriod = 0, maxClipDistances = 0, maxCullDistances = 0, maxCombinedClipAndCullDistances = 0, discreteQueuePriorities = 0, pointSizeRange = {0, 0}, lineWidthRange = {0, 0}, pointSizeGranularity = 0, lineWidthGranularity = 0, strictLines = 0, standardSampleLocations = 0, optimalBufferCopyOffsetAlignment = 0, optimalBufferCopyRowPitchAlignment = 0, nonCoherentAtomSize = 0}, sparseProperties = {residencyStandard2DBlockShape = 0, residencyStandard2DMultisampleBlockShape = 0, residencyStandard3DBlockShape = 0, residencyAlignedMipSize = 0, residencyNonResidentStrict = 0}}, queue_props = 0x0, memory_properties = {memoryTypeCount = 0, memoryTypes = {{propertyFlags = 0, heapIndex = 0} <repeats 32 times>}, memoryHeapCount = 0, memoryHeaps = {{size = 0, flags = 0} <repeats 16 times>}}, enabled_extension_count = 2, enabled_layer_count = 0, extension_names = {0x55ac44b33486 "VK_KHR_surface", 0x55ac44b33495 "VK_KHR_xcb_surface", 0x0 <repeats 62 times>}, enabled_layers = {0x0 <repeats 64 times>}, width = 0, height = 0, format = VK_FORMAT_UNDEFINED, color_space = VK_COLOR_SPACE_SRGB_NONLINEAR_KHR, fpGetPhysicalDeviceSurfaceSupportKHR = 0x0, fpGetPhysicalDeviceSurfaceCapabilitiesKHR = 0x0, fpGetPhysicalDeviceSurfaceFormatsKHR = 0x0, fpGetPhysicalDeviceSurfacePresentModesKHR = 0x0, fpCreateSwapchainKHR = 0x0, fpDestroySwapchainKHR = 0x0, fpGetSwapchainImagesKHR = 0x0, fpAcquireNextImageKHR = 0x0, fpQueuePresentKHR = 0x0, fpGetRefreshCycleDurationGOOGLE = 0x0, fpGetPastPresentationTimingGOOGLE = 0x0, swapchainImageCount = 0, swapchain = 0x0, swapchain_image_resources = 0x0, presentMode = VK_PRESENT_MODE_FIFO_KHR, fences = {0x0, 0x0}, frame_index = 0, cmd_pool = 0x0, present_cmd_pool = 0x0, depth = {format = VK_FORMAT_UNDEFINED, image = 0x0, mem_alloc = {sType = VK_STRUCTURE_TYPE_APPLICATION_INFO, pNext = 0x0, allocationSize = 0, memoryTypeIndex = 0}, mem = 0x0, view = 0x0}, textures = {{sampler = 0x0, image = 0x0, buffer = 0x0, imageLayout = VK_IMAGE_LAYOUT_UNDEFINED, mem_alloc = {sType = VK_STRUCTURE_TYPE_APPLICATION_INFO, pNext = 0x0, allocationSize = 0, memoryTypeIndex = 0}, mem = 0x0, view = 0x0, tex_width = 0, tex_height = 0}}, staging_texture = {sampler = 0x0, image = 0x0, buffer = 0x0, imageLayout = VK_IMAGE_LAYOUT_UNDEFINED, mem_alloc = {sType = VK_STRUCTURE_TYPE_APPLICATION_INFO, pNext = 0x0, allocationSize = 0, memoryTypeIndex = 0}, mem = 0x0, view = 0x0, tex_width = 0, tex_height = 0}, cmd = 0x0, pipeline_layout = 0x0, desc_layout = 0x0, pipelineCache = 0x0, render_pass = 0x0, pipeline = 0x0, projection_matrix = {{0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}}, view_matrix = {{0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}}, model_matrix = {{0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 0, 0}}, spin_angle = 0, spin_increment =0, pause = false, vert_shader_module = 0x0, frag_shader_module = 0x0, desc_pool = 0x0, quit = false, curFrame = 0, frameCount = 2147483647, validate = false, validate_checks_disabled = false, use_break = false, suppress_popups = false, CreateDebugUtilsMessengerEXT = 0x0, DestroyDebugUtilsMessengerEXT = 0x0, SubmitDebugUtilsMessageEXT = 0x0, CmdBeginDebugUtilsLabelEXT = 0x0, CmdEndDebugUtilsLabelEXT = 0x0, CmdInsertDebugUtilsLabelEXT = 0x0, SetDebugUtilsObjectNameEXT = 0x0, dbg_messenger = 0x0, current_buffer = 0, queue_family_count = 0}
```

**Comment by ZaWertun** [comment]
Maybe there is a crash while reading CPU info? Or something like that?

**Comment by flgx16** [comment]
That should be fixed on master branch, but not sure why it happened with default settings

**Comment by ZaWertun** [comment]
Thanks for quick response! :)
Should I close issue now or wait while new version with fix is released?

**Comment by flgx16** [comment]
Probably wait

**Comment by jackun** [comment]
@ZaWertun does [this fix it?](https://github.com/flightlessmango/MangoHud/suites/1578195258/artifacts/28529369)

**Comment by ZaWertun** [comment]
> @ZaWertun does [this fix it?](https://github.com/flightlessmango/MangoHud/suites/1578195258/artifacts/28529369)

I'll try it a bit later.

**Comment by ZaWertun** [comment]
> @ZaWertun does [this fix it?](https://github.com/flightlessmango/MangoHud/suites/1578195258/artifacts/28529369)

Yep, it works!

![изображение](https://user-images.githubusercontent.com/1080298/100547104-0d8ff500-3276-11eb-9a50-e4b1d92fc1f1.png)

**Comment by ZaWertun** [comment]
Fixed in version 0.6.1 (https://github.com/flightlessmango/MangoHud/releases/tag/v0.6.1).
Thanks to everyone!
